internal/api/grpc/management/user.go

```diff
@@ -68,7 +68,7 @@ func (s *Server) ListUsers(ctx context.Context, req *mgmt_pb.ListUsersRequest) (
 	if err != nil {
 		return nil, err
 	}
-	res, err := s.query.SearchUsers(ctx, queries, nil)
+	res, err := s.query.SearchUsers(ctx, queries, "", nil)
 	if err != nil {
```

Comment:
The orgID is available in the context, as used above. Let's assign is to a variable and pass it to the function:

```go
orgID := authz.GetCtxData(ctx).OrgID
	err = queries.AppendMyResourceOwnerQuery(orgID)
	if err != nil {
		return nil, err
	}
	res, err := s.query.SearchUsers(ctx, queries, orgID, nil)
```